src/main/java/edu/northeastern/truthtree/controller/collections/Collections.java

```diff
@@ -19,11 +29,25 @@ public Collections(ICollectionsService service) {
 		this.service = service;
 	}
 
+	/**
+	 * Represents collections interface. With an optional query parameter of
+	 * locationId, user can search for all the collections in the TruthTree
+	 * Application. If user provides locationId, additionally, this api would filter
+	 * out collections relevant to that location.
+	 * 
+	 * @param locationId
+	 *            which represents unique identifier for a location.
+	 * 
+	 * @author nehashukla
+	 *
+	 */
 	@Override
-	@RequestMapping("/api/collections")
-	public String getCollections(@RequestParam(name = "locationId", required = false) Integer locationId) {
```

Comment:
I recommending changed the return type to JSONArray or Object instead of String. This way it will appear, on swagger and API, as a JSON with character returns instead of a single line String.

Reply:
That's what included in this PR. I changed String return type to Object.

Reply:
It would help if I looked at the right color haha my bad.

Reply:
No probs 💃